tests/components/netatmo/test_api.py

```diff
@@ -1,14 +1,17 @@
 """The tests for the Netatmo oauth2 api."""
```

Comment:
This test should be part of an `test_init.py` module. Please move it.

Reply:
Ah, sure, no problem.